src/main/java/katsu/parser/Parser.java

```diff
@@ -0,0 +1,88 @@
+package katsu.parser;
+
+import katsu.Katsu;
+import katsu.ui.UI;
+
+/**
+ * Handles parsing and processing of user commands for the Katsu application.
+ * Translates user input into appropriate actions for the bot.
+ */
+public class Parser {
+
+    /**
+     * Processes and executes the user command by delegating to appropriate bot methods.
+     *
+     * @param order the full command string entered by the user
+     * @param bot the Katsu bot instance to execute commands on
+     */
+    public static void handleCommand(String order, Katsu bot) {
+        if (order.isEmpty()) {
+            System.out.println(UI.SEPARATOR);
+            System.out.println(UI.INDENT + "Quack! You need to write something!\n");
+            System.out.println(UI.INDENT + "Here is some commands to help you:");
+            Katsu.allCommands();
+            System.out.println(UI.SEPARATOR + "\n");
+            return;
+        }
+
+        // split words by empty space
+        String[] words = order.split(" ");
+        System.out.println(UI.SEPARATOR);
+
+        switch (words[0]) {
+        case "bye":
+            bot.deactivate();
+            break;
+        case "list":
+        case "ls":
```

Comment:
```suggestion
        case "list", "ls":
```
Without this, will have to add an explicit `// fallthrough` comment to follow coding standard.